cem/add-dependencies-in-description.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * CEM analyzer plugin: add-dependencies-in-description
+ *
+ * This plugin adds the dependencies and the dependants at the description of each component if they have some.
+ */
+import {
+  getComponentsGraph,
+  getComponentsTree,
+} from '../tasks/component-usage-graph.js';
+
+const graph = await getComponentsGraph();
+
+export default function addDependenciesInDescription () {
+
+  return {
+    name: 'add-dependencies-in-description',
+    moduleLinkPhase ({ moduleDoc }) {
+
+      const isSmartComponentFile = !moduleDoc.path.includes('smart');
```

Comment:
**suggestion:** `smart` could be used within the name of the component, for instance `cc-smart-form`. It's unlikely but just in case I would go for a more specific test:

```suggestion
      const isSmartComponentFile = !moduleDoc.path.includes('.smart.js');
```

Reply:
**question:** I'm confused because the const name is `isSmartComponentFile`, meaning that it should be true if the component file is for a smart component but the value indicates the contrary :thinking:

Reply:
> question: I'm confused because the const name is isSmartComponentFile, meaning that it should be true if the component file is for a smart component but the value indicates the contrary 🤔

Oops, it's a mistake, nice catch :wink: 

> suggestion: smart could be used within the name of the component, for instance cc-smart-form. It's unlikely but just in case I would go for a more specific test:
Will do the change :wink: